config/locales/models/en.yml

```diff
@@ -96,6 +98,7 @@ en:
           format: "%{message}"
         comment:
           attributes:
+            name: 
```

Comment:
Is this line necessary?

Reply:
Nope! I bet it's from one of my earlier, failed experiments. I've cleaned it up